maps/tether/tether_defines.dm

```diff
@@ -155,6 +155,12 @@
 		/area/tether/surfacebase/emergency_storage/rnd,
 		/area/tether/surfacebase/emergency_storage/atrium)
 
+//Medical Move
+	unit_test_exempt_from_atmos = list(
+		/area/tether/surfacebase/lowernortheva, // it airlock
+		/area/tether/surfacebase/lowernortheva/external //it outside
```

Comment:
This second line is unneeded as by default, anything a child of lowernortheva will inherit the exemption, IIRC.

Reply:
Alsooooooo, can, uhhh, remove my TFF comment since it no longer is relevant, just FYI.